zerver/models.py

```diff
@@ -2868,3 +2865,9 @@ def get_fake_email_domain() -> str:
         raise InvalidFakeEmailDomain(settings.FAKE_EMAIL_DOMAIN + ' is not a valid domain.')
 
     return settings.FAKE_EMAIL_DOMAIN
+
+class AlertWords(models.Model):
+    user_profile = models.ForeignKey(UserProfile, on_delete=CASCADE)  # type: UserProfile
+    word = models.TextField()   # type: str
+    DEFAULT_ALERT_WORD_COLOR = u"hsla(102, 85%, 57%, 0.5)"
+    color = models.TextField(default = DEFAULT_ALERT_WORD_COLOR)   # type: str
```

Comment:
Let's not add the `color` table in this migration; we should first do a commit where all it does is extracts the `AlertWord` table (note the name should not be plural!) with no functional changes (this part is a good idea regardless of the merits of the other feature).  And then we can add a smaller commit to add the new color feature.